### PR TITLE
3.1 version OHOS adaptation

### PR DESCRIPTION
**Pull request by IceBBBBB**

### CODE REVIEW

File: tests/lua-tests/project/Classes/AppDelegate.cpp [hunk 1]
```diff
@@ -47,13 +47,14 @@ bool AppDelegate::applicationDidFinishLaunching()
...
-
-#if (CC_TARGET_PLATFORM == CC_PLATFORM_WIN32 || CC_TARGET_PLATFORM == CC_PLATFORM_ANDROID ||CC_TARGET_PLATFORM == CC_PLATFORM_IOS || CC_TARGET_PLATFORM == CC_PLATFORM_MAC)
+    
+#if (CC_TARGET_PLATFORM == CC_PLATFORM_WIN32 || CC_TARGET_PLATFORM == CC_PLATFORM_ANDROID ||CC_TARGET_PLATFORM == CC_PLATFORM_IOS || CC_TARGET_PLATFORM == CC_PLATFORM_MAC || CC_TARGET_PLATFORM == CC_PLATFORM_OHOS)
     LuaStack* stack = pEngine->getLuaStack();
     register_assetsmanager_test_sample(stack->getLuaState());
 #endif
 
-    pEngine->executeScriptFile("src/controller.lua");
+    FileUtils::getInstance()->addSearchPath("");
```
Comment (minggo) [inline tests/lua-tests/project/Classes/AppDelegate.cpp]: Why need add an empty path?

File: tests/lua-tests/src/controller.lua [hunk 2]
```diff
@@ -3,7 +3,7 @@
 collectgarbage("setpause", 100) 
 collectgarbage("setstepmul", 5000)
 	
-require "src/mainMenu"
+require "mainMenu"
```
Comment (minggo) [inline tests/lua-tests/src/controller.lua]: Why change path? Is it a problem?

File: tests/lua-tests/src/controller.lua [hunk 3]
```diff
@@ -62,6 +62,7 @@ else
 end
 
 fileUtils:setSearchPaths(searchPaths)
+fileUtils:setSearchPaths("")
```
Comment (minggo) [inline tests/lua-tests/src/controller.lua]: Why add an empty path?

File: tests/lua-tests/src/helper.lua [hunk 4]
```diff
@@ -1,4 +1,4 @@
-require "Cocos2d"
+require "cocos/Cocos2d"
```
Comment (minggo) [inline tests/lua-tests/src/helper.lua]: Why change path?

Comment (minggo) [reply tests/lua-tests/src/helper.lua]: It seems many codes changes the path, i don't list all the modifcations.

File: tests/lua-tests/project/Classes/AppDelegate.cpp (hunk 1 above)
Comment (IceBBBBB) [reply tests/lua-tests/project/Classes/AppDelegate.cpp]: modified

File: tests/lua-tests/src/controller.lua (hunk 2 above)
Comment (IceBBBBB) [reply tests/lua-tests/src/controller.lua]: modified

File: tests/lua-tests/src/controller.lua (hunk 3 above)
Comment (IceBBBBB) [reply tests/lua-tests/src/controller.lua]: modified

File: tests/lua-tests/src/helper.lua (hunk 4 above)
Comment (IceBBBBB) [reply tests/lua-tests/src/helper.lua]: modified

**Pull request merged**